packages/features/auth/lib/getServerSession.ts

```diff
@@ -46,7 +46,7 @@ export async function getServerSession(options: {
     return cachedSession;
   }
 
-  const user = await prisma.user.findUnique({
+  const user = await prisma.user.findFirst({
```

Comment:
Why are we doing a findFirst. emails is still unique field. Looks like a change from one of the org branches that was reverted before going to main.

Reply:
Merged with `main`